recipes/hyperspy_gui_ipywidgets/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hyperspy-gui-ipywidgets" %}
+{% set filename = "hyperspy_gui_ipywidgets" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "31803a08d74493b697714ca759a217568fda08a33dad3d39dfd9bf3bf0923fbc" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - link-traits
+    - hyperspy >=1.3
+
+test:
+  imports:
+    - hyperspy_gui_ipywidgets
+
+  commands:
+    - export MPLBACKEND="agg"  # [unix]
+    - py.test --pyargs hyperspy_gui_ipywidgets  # [unix]
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/hyperspy/hyperspy_gui_ipywidgets
+  license: GPL v3
+  summary: "ipywidgets GUI elements for HyperSpy."
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
The LICENSE was already in the sdist package. I have just removed it from the recipe.